solar_installation/demo/im_livechat_channel.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <odoo noupdate="1">
     <record id="im_livechat.im_livechat_channel_data" model="im_livechat.channel">
-        <field name="website_published" eval="True"/>
+        <field name="is_published" eval="True"/>
         <field name="is_published" eval="True"/>
```

Comment:
Defined twice :eyes: and your test doesn't raise...

Reply:
I think the test does not raise because it is in the same definition, will add a new test for these cases